junit5/src/main/java/cz/xtf/junit5/listeners/ProjectCreator.java

```diff
@@ -19,6 +20,9 @@ public void testPlanExecutionStarted(TestPlan testPlan) {
 		if(openShift.getProject() == null) {
 			openShift.createProjectRequest();
 		}
+		if(OpenShiftConfig.pullSecret() != null) {
```

Comment:
also here, 

I suspect there may be a race between the project being created and the creation of the secret in the namespace. ( openShift.createProjectRequest(); is async afaik, so I suspect the creation may fail if the namespace isn't created by then,)